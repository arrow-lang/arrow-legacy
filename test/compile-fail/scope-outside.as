let a = 20;

{
  let b = 30;
}

let c = b;
