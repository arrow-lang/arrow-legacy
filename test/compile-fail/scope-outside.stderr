[0;37mtest/compile-fail/scope-outside.as:7.9-10: [0;31merror: [1;37muse of unresolved name 'b'[0m
